config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get '/', to: 'user#index'
+  get '/users/:id', to: 'user#show'
+  get '/users/:id/posts', to: 'posts#index'
+  get '/users/:id/posts/:postId', to: 'posts#show'
```

Comment:
- [ ] I suggest that you use Resource routing it allows you to quickly declare all of the common routes for a given resourceful controller you can simply do
`resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
`